lib/algolia/analytics.rb

```diff
@@ -0,0 +1,77 @@
+module Algolia
+
+  class Analytics
+    attr_reader :ssl, :ssl_version, :headers
+
+    def initialize(client, params)
+      @client = client
+
+      @headers      = params[:headers]
+      @ssl          = params[:ssl].nil? ? true : params[:ssl]
```

Comment:
It is easier to read: `@ssl = params[:ssl] || true`

Reply:
👍 definitely!
I copy-pasted the init function of the client 😅

Reply:
Yep, it's not necessary a reference 😅